vulnfeeds/cves/versions.go

```diff
@@ -37,6 +38,21 @@ type VersionInfo struct {
 	AffectedVersions []AffectedVersion
 }
 
+type CPE struct {
+	CPEVersion string
+	Part       string
+	Vendor     string
+	Product    string
+	Version    string
+	Update     string
+	Edition    string
+	Language   string
+	SWEdition  string
+	TargetSw   string
```

Comment:
Nit: s/Sw/SW/ on this line and next.

See https://github.com/golang/go/wiki/CodeReviewComments

Reply:
Done.

Reply:
(meant to link directly to https://github.com/golang/go/wiki/CodeReviewComments#initialisms)